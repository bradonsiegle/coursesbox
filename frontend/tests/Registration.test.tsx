import { render } from '@/test-utils';
import Registration from '@/pages/registration';

describe('Registration page', () => {
	it('should render', () => {
		const { container } = render(<Registration />);
		expect(container).toMatchSnapshot();
	});
});
