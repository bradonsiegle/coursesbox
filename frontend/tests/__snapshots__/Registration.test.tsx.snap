// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Registration page Client validation check 1`] = `
<span
  class="css-1reg724"
  role="alert"
>
   
</span>
`;

exports[`Registration page Client validation check 2`] = `
<span
  class="css-1reg724"
  role="alert"
>
  <span
    class="css-sk8pyf"
  >
    Email is invalid
  </span>
</span>
`;

exports[`Registration page Client validation check 3`] = `
<span
  class="css-1reg724"
  role="alert"
>
   
</span>
`;

exports[`Registration page Render check 1`] = `
<div>
  <form>
    <div
      class="css-xlq2o8"
    >
      <section
        class="css-vpzuu8"
      >
        <h2>
          Create an account
        </h2>
        <h3>
           
        </h3>
        <label
          class="css-1805hjr"
          height="7"
          width="20"
        >
          <span
            class="css-1grri41"
          >
            username
          </span>
          <div
            class="css-52ovgc"
          >
            <input
              class="css-1xhyq1j"
              name="username"
              placeholder="username"
            />
          </div>
          <span
            class="css-1reg724"
            role="alert"
          >
             
          </span>
        </label>
        <label
          class="css-1805hjr"
          height="7"
          width="20"
        >
          <span
            class="css-1grri41"
          >
            email
          </span>
          <div
            class="css-52ovgc"
          >
            <input
              class="css-1xhyq1j"
              name="email"
              placeholder="email"
              type="email"
            />
          </div>
          <span
            class="css-1reg724"
            role="alert"
          >
             
          </span>
        </label>
        <label
          class="css-1805hjr"
          height="7"
          width="20"
        >
          <span
            class="css-1grri41"
          >
            password
          </span>
          <div
            class="css-52ovgc"
          >
            <input
              class="css-1xhyq1j"
              name="password"
              placeholder="password"
              role="textbox"
              type="password"
            />
          </div>
          <span
            class="css-1reg724"
            role="alert"
          >
             
          </span>
        </label>
        <button
          class="css-4dagrr"
          color="primary"
          type="submit"
        >
          Sign Up
        </button>
        <h3>
          Already have an account?
           
          <a
            class="css-137tmhu"
            href="/login"
          >
            Login
          </a>
        </h3>
      </section>
    </div>
  </form>
</div>
`;
